clang/include/clang/CConv/ProgramInfo.h

```diff
@@ -117,6 +117,11 @@ class ProgramInfo : public ProgramVariableAdder {
   void constrainWildIfMacro(ConstraintVariable *CV, SourceLocation Location,
                             PersistentSourceLoc *PSL = nullptr);
 
+  std::map<PersistentSourceLoc, std::pair<CVarSet, bool>> typedefVars;
```

Comment:
Is this meant to be a non-`private` map? Typically we would make it private and then return it as `const`. See the `getVarMap` method above.